en/advanced-topics/dynamic-atlas.md

```diff
@@ -32,6 +37,18 @@ cc.dynamicAtlasManager.maxFrameSize = 512;
 
 You can refer to the API docs [DynamicAtlasManager](../../../api/en/classes/DynamicAtlasManager.html) for details.
 
+## Support custom rendering components
+
+Currently, the Dynamic Atlas Manager only supports the pack of the Sprite render component. If you want to add support for other types of render components, you can customize the rendering component yourself:
```

Comment:
Currently, the Dynamic Atlas Manager only supports some builtin rendering components such as Sprite or Label. If you want to add support for other component types, you can customize a rendering component yourself: